swagger.json

```diff
@@ -2574,9 +2574,234 @@
                     }
                 }
             }
+        },
+        "/addresses": {
+            "get": {
+                "tags": [
+                    "Default", "Fleet"
+                ],
+                "summary": "/addresses",
+                "description": "Fetch all addresses/geofences for the organization",
+                "operationId": "GetOrganizationAddresses",
+                "parameters": [
+                    { "$ref": "#/parameters/accessTokenParam" }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "List of addresses/geofences. For each address entry, the geofence field will only be populated with either 'circle' or 'polygon', depending on the geofence type.",
+                        "schema": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#definitions/Address"
+                            }
+                        }
+                    },
+                    "default": {
+                        "description": "Unexpected error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "post": {
+                "tags": [
+                    "Default", "Fleet"
+                ],
+                "summary": "/addresses",
+                "description": "Add one or more addresses to the organization",
+                "operationId": "AddOrganizationAddresses",
+                "parameters": [
+                    { "$ref": "#/parameters/accessTokenParam" },
+                    { "$ref": "#/parameters/addressAddParam" }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Addresses were successfully added. No response body is returned."
```

Comment:
We should return the ID of the newly created address (or preferably the entire address payload).

Reply:
Is there a guideline on what we expect for write action responses? I chose not to return much information to replicate the current responses of our models/controllers code. 
As a side note, I wish that this kind of comment come earlier in the code review or api spec review - I don’t think I did a good job of conveying expectations for the post/patch responses :(

Reply:
Agreed, I should have caught this during the implementation code review. From a customer's perspective, they would need the newly created address id in order to do anything useful with the new address.

Reply:
We can always fix this in a follow-up PR.

Reply:
This PR is about documenting the code, so we don't need to address here. However, we should definitely start to return the created object in the response payload. Otherwise this API is a black hole and the customer needs to search for the created address id.

Reply:
Absolutely. Let's update the "Go ❤️ APIs @ 🦉" doc to have some basic best practices for each http verb.

Reply:
Since this would likely map from the mutation response, should that necessarily be an API design principle or a general graphql best practice?